gitlab/__init__.py

```diff
@@ -586,27 +586,25 @@ def getprojecthook(self, project_id, hook_id):
         else:
             return False
 
-    def addprojecthook(self, project_id, url, push=False, issues=False, merge_requests=False, tag_push=False):
+    def addprojecthook(self, project_id, url, **kwargs):
         """
         add a hook to a project
         :param id_: project id
         :param url: url of the hook
         :return: True if success
         """
         data = {"id": project_id, "url": url}
-        data['push_events'] = int(bool(push))
-        data['issues_events'] = int(bool(issues))
-        data['merge_requests_events'] = int(bool(merge_requests))
-        data['tag_push_events'] = int(bool(tag_push))
+        if kwargs:
+            data.update(kwargs)
```

Comment:
Doesn't this change the data sent over the wire from "0/1" (int) to "False/True"?